sdk/python/kfp/v2/dsl/artifacts_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for kfp.v2.ds.artifacts module."""
+import unittest
+from kfp.v2.dsl import artifact
+from kfp.v2.dsl import artifacts
```

Comment:
nit: Just realized the two module names `artifact` and `artifacts` are very close. Maybe `artifacts` can be something like `mlmd_artifacts` or `system_artifacts`? Up to you.

Reply:
Renamed to `ontology_artifacts`